src/ynViewer.sol

```diff
@@ -41,8 +41,9 @@ contract ynViewer is IynViewer {
     function withdrawalDelayBlocks(address _strategy) external view returns (uint256) {
         IDelegationManager _delegationManager = stakingNodesManager.delegationManager();
         uint256 _minDelay = _delegationManager.minWithdrawalDelayBlocks();
-        uint256 _strategyDelay = _delegationManager.strategyWithdrawalDelayBlocks(IStrategy(_strategy));
-        return _minDelay > _strategyDelay ? _minDelay : _strategyDelay;
+        // uint256 _strategyDelay = _delegationManager.__deprecated_strategyWithdrawalDelayBlocks(IStrategy(_strategy));
```

Comment:
why is this commented out?

Final code should not have commented out blocks like this, they are either useful or they are removed